mautic/mautic
# Fix including/excluding operator in dwc

| Q                                      | A
| -------------------------------------- | ---
| Branch?                                | 3.2 for bug fixes
| Bug fix?                               | yes
| New feature?                           | no
| Deprecations?                          | no
| BC breaks?                             | no
| Automated tests included?              | yes
| Related user documentation PR URL      | 
| Related developer documentation PR URL | 
| Issue(s) addressed                     | 

#### Description:
DWC filter is not matching in operator for countries.
Discovered when fixing https://github.com/mautic/mautic/pull/9519
Reproduction steps are almost same

#### Steps to test this PR:
0. Load up [this PR](https://mautibox.com)1. Create focus item and publish it
2. Create DWC
 - Use content 9581 DWC content {focus=1} where 1 is id of your created focus item
 - Is campaign based = no
 - Requested Slot Name = slot1
 - Add filter - Country including Czech Republic
3. Create landing page
 - with dynamic content where
 - Slot name is slot1
 - Slot content is Dynamic web content applied {dwc=slot1}‍
4. Use incognito mode in browser
 - add contact via form in incognito mode - contact should be added to CS
 - try to show page created in step 3 in incognito mode - focus item is not displayed
 - in CS set country to Czech Republic for this contact
 - reload page in incognito mode - focus item is not displayed - JS is not loaded

#### Steps to test this PR:
1. Use reproduction steps
2. Focus item is displayed and JS is loaded and operator is working correctly
